metafix/src/test/resources/org/metafacture/metafix/integration/method/fromJson/toJson/lookupInArrayOfObjects/test.fix

```diff
@@ -0,0 +1,9 @@
+put_map("map",
+  "dog":"mammal",
+  "cat":"mammal",
+  "parrot":"bird",
+  "shark":"fish",
+  "dragon":"ficational animal",
+  "unicorn":"ficational animal")
```

Comment:
I know, these are "only" test files, but I'd really like to not have typos in those either.